ush/forecast_predet.sh

```diff
@@ -265,4 +265,6 @@
 
   if [[ ! -d "${COM_CHEM_HISTORY}" ]]; then mkdir -p "${COM_CHEM_HISTORY}"; fi
 
+  GOCART_OUTPUT_FH=$(seq -s ' ' "${FHMIN}" "6" "${FHMAX}")
```

Comment:
Since this script is always sourced, can we try putting
```
# shellcheck disable=SC2034
```
at the top of the script (directly after the shebang) and remove the other instances?